src/Codeception/Lib/Driver/Db.php

```diff
@@ -221,7 +221,20 @@ public function executeQuery($query, array $params)
             throw new \Exception("Query '$query' can't be prepared.");
         }
 
-        $sth->execute($params);
+        $i = 0;
+        foreach ($params as $value) {
+            $i++;
+            if (is_bool($value)) {
+                $type = \PDO::PARAM_BOOL;
```

Comment:
Added a condition for boolean as well. This *only* handles literal booleans because that is all that can be safely assumed.